src/Pages/firebase.js

```diff
@@ -0,0 +1,64 @@
+import { db } from '../lib/firebase';
+import { collection, getDocs, addDoc, doc } from 'firebase/firestore';
+import { useEffect, useState } from 'react';
+
+export default function Firebase() {
+  // const [item, setItem] = useState('');
+  const [docs, setDocs] = useState([]);
+
+  useEffect(() => {
+    const getGrocery = async () => {
+      try {
+        const querySnapshot = await getDocs(collection(db, 'groceries'));
+        const snapshotDocs = [];
+        //loop through snapshot & push to array
+        querySnapshot.forEach((doc) => snapshotDocs.push(doc.data()));
```

Comment:
Nit and mostly stylistic...any time you find yourself combining `.forEach` with a `.push` to an empty array, you can usually use `.map`. 

The main reason why this would be preferable in the eyes of some is just because it's more of a "functional programming" approach which sees any mutation as a mortal sin. ;-P. Either way works just fine though, but just thought that I'd point it out.